lib/actual_db_schema/engine.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module ActualDbSchema
+  # It isolates the namespace to avoid conflicts with the main application.
+  class Engine < ::Rails::Engine
+    isolate_namespace ActualDbSchema
+
+    initializer "actual_db_schema.append_routes", after: "add_routing_paths" do |app|
+      app.routes.append do
+        mount ActualDbSchema::Engine => "/actual_db_schema"
```

Comment:
```suggestion
        mount ActualDbSchema::Engine => "/rails"
```

Can we do this? Won't it conflict with rails mailers previews?